### PR TITLE
Siphon cleanup

### LINKED ISSUE #30: AMS - Verify/Update Siphon Notebook
Go through the siphon notebook and do any needed cleanup/updating before AMS.

### PR DESCRIPTION
**Pull request by jrleeman**
Closes #30 by doing some housekeeping in the siphon notebook (and finding some siphon pain points!).

### CODE REVIEW

File: pages/workshop/Siphon/Siphon Overview.ipynb [hunk 1]
```diff
@@ -187,20 +207,24 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
    "source": [
-    "Accessing catalogs is only part of the story; Siphon is much more useful if you're trying to access/download datasets.\n",
-    "\n",
-    "For instance, using our data that we just retrieved:"
+    "# Solution from above in case you had trouble\n",
+    "date = datetime.utcnow() - timedelta(days=1)\n",
+    "cat = TDSCatalog(f'http://thredds.ucar.edu/thredds/catalog/satellite/SFC-T/SUPER-NATIONAL_1km/{date:%Y%m%d}/catalog.xml')\n",
```
Comment (dopplershift) [inline pages/workshop/Siphon/Siphon Overview.ipynb]: While we're here, is there a less esoteric dataset we could grab? Given that the GINI products are on their way out...

Comment (jrleeman) [reply pages/workshop/Siphon/Siphon Overview.ipynb]: Soooo.. I tried that thinking I'll get a GOES product- like sea surface temps. Filtering on a time range appears to not work with the new file name convention?

Comment (dopplershift) [reply pages/workshop/Siphon/Siphon Overview.ipynb]: It should be possible to send the appropriate regex and date format string, but that's probably not a great thing to use in a light intro. Or maybe it is?

Comment (jrleeman) [reply pages/workshop/Siphon/Siphon Overview.ipynb]: Well, I'd say regex != light intro, but regex == reality. Maybe we'll give it a shot?

Comment (dopplershift) [reply pages/workshop/Siphon/Siphon Overview.ipynb]: Sure.

Comment (jrleeman) [reply pages/workshop/Siphon/Siphon Overview.ipynb]: Strike 2 - no julian day support the way that setup currently works.

Comment (dopplershift) [reply pages/workshop/Siphon/Siphon Overview.ipynb]: *sigh* That's because that support is sitting in master. 😠 Ugh.  Level2?

**Comment by jrleeman** [comment]
Failure unrelated - remote data access for the win.

**Pull request merged**